### PR TITLE
 Resolved the word 'du', resolved for tuesday the 13th of the 6th mon…

### PR DESCRIPTION
**Pull request by CiaraCrowe**
Resolved for the french word 'du', meaning of/of the. 
Resolved fr the french word 'mardi 13 du 6e mois'. 
Updated versions

### CODE REVIEW

File: Python/libraries/datatypes-timex-expression/setup.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def read(fname):
 
 
 NAME = 'datatypes_timex_expression_genesys'
-VERSION = '1.1.25'
+VERSION = '1.1.25a0'
```
Comment (samhickey25) [inline Python/libraries/datatypes-timex-expression/setup.py]: You should go to the next value, so this should be `1.1.26a0`

**Comment by CiaraCrowe** [comment]
https://jenkins.ininica.com/view/Dialog%20Engine/job/NLU/job/Recognizers-Text/463/ 
Tests passing

**Review by Conor-Keaney — APPROVED** [review]

**Review by BobbyMartins5 — APPROVED** [review]

**Comment by kevinwalshgen** [comment]
> https://jenkins.ininica.com/view/Dialog%20Engine/job/NLU/job/Recognizers-Text/463/ Tests passing

Were these ran after updating the version number?

File: Specs/DateTime/French/DateExtractor.json
Comment (kevinwalshgen) [inline Specs/DateTime/French/DateExtractor.json:1]: Any of the other existing tests, currently skipped, passing with this change?

Comment (CiaraCrowe) [reply Specs/DateTime/French/DateExtractor.json:1]: Not the skipped ones

**Comment by CiaraCrowe** [comment]
https://jenkins.ininica.com/view/Dialog%20Engine/job/NLU/job/Recognizers-Text/465/ final tests passing

**Review by KanchanKumar12 — APPROVED** [review]

**Review by colm-dillon — APPROVED** [review]
Nice fix Ciara.

**Review by andrew-gradinari — APPROVED** [review]

**Review by samhickey25 — APPROVED** [review]

**Pull request merged**